consensus/consensus_viewchange_msg.go

```diff
@@ -35,14 +37,21 @@ func (consensus *Consensus) constructViewChangeMessage(pubKey *bls.PublicKey, pr
 	)
 	preparedMsg := consensus.FBFTLog.FindMessageByMaxViewID(preparedMsgs)
 
+	block := consensus.FBFTLog.GetBlockByHash(preparedMsg.BlockHash)
 	var msgToSign []byte
-	if preparedMsg == nil {
+
+	encodedBlock, err := rlp.EncodeToBytes(block)
+	if err != nil {
+		consensus.getLogger().Err(err).Msg("[constructViewChangeMessage] Failed encoding block")
```

Comment:
Please use plain global logger

Reply:
I will leave it for your batch update (to be consistent with existing code for now)